e2etest/testcases.json

```diff
@@ -54,5 +54,17 @@
 	{
 		"case_name": "newrelic_exporter_metric_mock",
 		"platforms": ["EC2", "ECS", "EKS", "LOCAL", "SOAKING", "NEG_SOAKING"]
+	},
+	{
+		"case_name": "prometheus_mock",
+		"platforms": ["EC2", "EKS", "LOCAL", "SOAKING"]
+	},
+	{
+		"case_name": "prometheus_static",
```

Comment:
if you wanna run real endpoint test on soaking and perf test, you want add "SOAKING" here and for prometheus_sd

Reply:
Yup, I think we'll leave out soaking and performance tests for static and sd for now. Prometheus_mock does pretty much the same thing right now but exports to the mocked_server.

Reply:
prometheus_sd we definitely have to leave out of perf/soaking because it is very specifically testing EKS service discovery and won't run on EC2.